src/server/index.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+require('dotenv').config();
+
+const WebSocket = require('ws');
+const roomsController = require('./controllers/rooms.controller.js');
+
+const PORT = process.env.PORT || 3001;
+
+const wss = new WebSocket.Server({ port: PORT });
+
+wss.on('connection', (ws) => {
+  ws.on('message', (message) => {
+    let data;
+
+    try {
+      data = JSON.parse(message);
+    } catch (error) {
+      ws.send(
+        JSON.stringify({ type: 'error', message: 'Invalid JSON format' }),
+      );
+    }
```

Comment:
After catching an error in JSON parsing, it's a good practice to return immediately to prevent further execution with an undefined `data` variable.